lib/CModHandler.h

```diff
@@ -382,6 +384,8 @@ class DLL_LINKAGE CModHandler
 			h & ATTACK_POINTS_DMG_MULTIPLIER_CAP;
 			h & DEFENSE_POINT_DMG_MULTIPLIER;
 			h & DEFENSE_POINTS_DMG_MULTIPLIER_CAP;
+			h & NEW_HERO_ALWAYS_3_CREATURE_STACKS;
+			h & DEFAULT_TOWN_ALWAYS_2_DWELLINGS;

```

Comment:
please, bump serialisation version and also you can provide backward compatibility easy here:
```suggestion
		if(version >= xxx)
		{
			h & NEW_HERO_ALWAYS_3_CREATURE_STACKS;
			h & DEFAULT_TOWN_ALWAYS_2_DWELLINGS;
		}
```